server/src/routes/redeemRouter.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const { redeemCode } = require('../controllers');
+
+router.post('/', redeemCode);
+
+module.exports = router;
```

Comment:
why such small routers, combine the 2 into one `/promo` router, then do POST `/redeem` and POST `/`

Reply:
done